core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -54,7 +54,8 @@ function makeIdNumeric(id) {
 }
 
 function formatDate(date) {
-  return date.split(".")[0].replace("T", " ");
+  if (date <= 0) return "-";
```

Comment:
I see. But to me, blank indicates that something is wrong. "-" is a little better. So I changed it back to "-".